config/runtime/kunit.jinja2

```diff
@@ -26,6 +26,17 @@ ARCH = '{{ arch }}'
 
 {% block python_job -%}
 class Job(BaseJob):
+    def _evaluate_job_node_result(self, child_nodes):
+        result = None
+        result_list = [child['node']['result'] for child in child_nodes]
+        if 'fail' in result_list:
+            result = 'fail'
+        elif all(result == 'pass' for result in result_list) or 'pass' in result_list:
+            result = 'pass'
+        elif all(result == 'skip' for result in result_list) or 'skip' in result_list:
+            result = 'skip'
+        return result
```

Comment:
This job result evaluation procedure is the same as the one [propsed for LAVA](https://github.com/kernelci/kernelci-core/blob/928b1f91b6714078affc1c0873083e7a75184b16/kernelci/runtime/lava.py#L167) - would it be possible to extract it to a common package and reuse it instead of duplicating?

Reply:
Yes, I can try that.

Reply:
Updated the code accordingly. 
Moved the method to `kernelci.runtime`.

Reply:
Tested OK on staging https://staging.kernelci.org:9000/viewer?node_id=6673de03638913d5a6abf4bd

Reply:
Looks good in https://github.com/kernelci/kernelci-core/pull/2571/files#diff-9ebe59937a7e94546a071f481a86fd3cd0ae41b4534c8cfe858eb4cf75f35b89R242 :+1: 

This method is now safe to be dropped from this patch